scipy/sparse/construct.py

```diff
@@ -714,6 +714,18 @@ def random(m, n, density=0.01, format='coo', dtype=None,
            [  0.,   0.,   0.,   0.],
            [  0.,   0.,  36.,   0.]])
 
+    >>> import numpy as np
```

Comment:
sorry for the late reply, i'm currently travelling - i agree, it would make the example a lot more readable that way. i will make the suggested changes once i'm back